internal/client/datadog/formatter_test.go

```diff
@@ -78,63 +79,17 @@ var _ = Describe("Formatter", func() {
 		Expect(string(helper.Decompress(result[0]))).To(ContainSubstring(`"points":[[0,9.000000],[0,1.000000]]`))
 	})
 
-	It("properly assigns all values to split results when splitting", func() {
-		// first test a scenario where we're not splitting as there's just one point
-		m := make(map[metric.MetricKey]metric.MetricValue)
-		m[metric.MetricKey{Name: "a"}] = metric.MetricValue{
-			Points: []metric.Point{{Value: 9}},
-			Tags:   []string{"some:tag", "other:tag"},
-			Host:   "some.host",
+	It("properly splits metrics into two maps", func() {
+		// now test a scenario where we're actually splitting metrics
+		for i := 0; i < 1000; i++ {
+			k, v := makeFakeMetric("metricName", 1000, uint64(i), defaultTags)
```

Comment:
need to modify how this is is constructed. Same as for the test in client_test.go